### ISSUE TITLE
This is still not fixed. Running grate on a postgres database which had previously been migrated using roundhouse does not work.

### ISSUE THREAD

**Issue by cocytus** [erikbra/grate#557]
This is still not fixed. Running grate on a postgres database which had previously been migrated using roundhouse does not work.

_Originally posted by @cocytus in https://github.com/erikbra/grate/issues/245#issuecomment-2228173931_

**Comment by cocytus** [comment]
Same issue as two years ago:
```
================================================================================
Grate Structure
================================================================================
An error occurred: Migration failed due to the following errors:

Update ("up"):
--------------------------------------------------------------------------------
01_version_add_status_column.sql: 42P01: relation "roundhouse_archive.Version" does not exist
(0,0): ALTER TABLE "{{SchemaName}}"."{{VersionTable}}"
      ^ 42P01: relation "roundhouse_archive.Version" does not exist
Exception: Database migration (Update) failed!, exiting.. 1
```

**Comment by erikbra** [comment]
This was a different error (though related). It is fixed, will be part of 1.8.
